alg/sha256_arm.c

```diff
@@ -98,8 +87,12 @@ SHA256_Transform_arm(uint32_t state[static restrict 8],
 #endif
 {
 	uint32x4_t Y[4];
+	uint32x4_t S0, S1; /* S0 and S1 alternate between ABEF and CDGH. */
```

Comment:
Have you benchmarked this vs. making it a two-element array?

Reply:
No... I can do, but I wouldn't expect that to make a difference...?

Reply:
I've seen that sort of thing confuse optimizers in the past.  If the performance is the same, I would prefer to have it as an array.

Reply:
Done, although of course we can't make that change until we're no longer passes `S` as an argument.